lib/skate/ueberauth/strategy/fake.ex

```diff
@@ -6,7 +6,7 @@ defmodule Skate.Ueberauth.Strategy.Fake do
   @impl Ueberauth.Strategy
   def handle_request!(conn) do
     conn
-    |> redirect!("/auth/cognito/callback")
+    |> redirect!("/auth/keycloak/callback")
```

Comment:
suggestion(non-blocking): can we use verified routes here?

Reply:
We can! c960b43d

Reply:
Cool! there's a few other places I noticed this url being used/updated without verified routes, I can go call them out if you'd like, but I consider this to be non blocking. So if you felt like updating them, I wouldn't say no 👀

Reply:
So we should definitely use verified routes for things like calls to `get`, what's your opinion on using it in assertions? Like should `assert redirected_to(conn) == "/foo"` become `assert redirected_to(conn) == ~p"/foo"`?

Reply:
(Sorry didn't see your comment till just now) If it's possible to use verified routes for assertions, I think it's worthwhile, as it attempts to convey that we intend the route to exist and therefore get compilation errors?